docs/intro/tutorial.rst

```diff
@@ -78,8 +79,8 @@ Our first Spider
 
 Spiders are classes that you define and that Scrapy uses to scrape information
 from a website (or a group of websites). They must subclass
-:class:`~scrapy.Spider` and define the initial requests to make,
-optionally how to follow links in the pages, and how to parse the downloaded
+:class:`~scrapy.Spider` and define the initial requests to be made.
+They can also optionally define how to follow page links and how to parse the downloaded
 page content to extract data.
```

Comment:
“They” here could now be understood as referring to requests, rather than spiders. And ”page links” can be interpreted as links to pages, rather than links from pages.